macros/snow-mask/get_meta_objects.sql

```diff
@@ -12,18 +12,30 @@
             {% if node_resource_type == "source" %} 
                 {% for column in columns if graph.sources[node_unique_id]['columns'][column]['meta'][meta_key] | length > 0 %}
                     {% set meta_dict = graph.sources[node_unique_id]['columns'][column]['meta'] %}
-                    {% for key, value in meta_dict.items() if key == meta_key %}
```

Comment:
This for loop was not needed - we can just select the key from the dict if it exists.